### PR TITLE
Dencun readiness - pin Lighthouse image tag in base Docker Compose setup

### PR DESCRIPTION
**Pull request by zakisaad**
Small tweak to the Docker Compose file, pinning Lighthouse to version 5.0.0. The upstream Lighthouse Docker hub repo does not update the `latest` tag whenever new images are pushed, thus the Docker Compose yaml (implicitly pulling `latest` due to missing a pin) pulls a very old version of Lighthouse.

**All lighthouse beacon nodes need to be updated to a minimum of 5.0.0 to be compatible with the upcoming Dencun release.** 

More info here: https://github.com/sigp/lighthouse/releases/tag/v5.0.0. TL;DR:

>  All Lighthouse Beacon Nodes and Validator Clients must be upgraded to a v5.x.x release to ensure they follow the correct chain.

As it's my first PR to this repo, keeping it very light. We've modified these base Docker Compose files in other ways internally, but this seems like a critical fix that should be included as a sane default in the base files before the imminent Dencun upgrade.

### CODE REVIEW

**Comment by rswanson** [comment]
This PR definitely would work as described, just wanted to clarify before approving though- @zakisaad what image do you get when you pull down `sigp/lighthouse:latest`? You mentioned its very old but I'm seeing a new `latest` tag pushed at the exact same time at the `5.0.0-modern` tag, if their `latest` on dockerhub is incorrect then we can give the lighthouse folks a heads up on that.

**Comment by zakisaad** [comment]
Hey @rswanson - I've just checked and you're right, the latest tag is indeed up to date (i.e. pulls down v5.0.0). It may have been that the `latest` tag went stale and wasn't updated on one of our previously configured machines.

**Comment by zakisaad** [comment]
Looks like a (related/variant?) of this issue hit Blast (L2) today as Dencun rolled out: https://twitter.com/Blast_L2/status/1767924021872455902

Their fix: https://github.com/blast-io/deployment/commit/66daab7abcf2d254166bf55731fd3d48fb8cf375

This is an option (`:latest` + always pull), but I feel that pinning specific versions is probably more prudent for stability. Backward incompatible changes/regressions could have unfavourable effects on a Reth node if a resync is required (downtime for x period).

**Comment by EurybiaWeb3** [comment]
> Small tweak to the Docker Compose file, pinning Lighthouse to version 5.0.0. The upstream Lighthouse Docker hub repo does not update the `latest` tag whenever new images are pushed, thus the Docker Compose yaml (implicitly pulling `latest` due to missing a pin) pulls a very old version of Lighthouse.
> 
> **All lighthouse beacon nodes need to be updated to a minimum of 5.0.0 to be compatible with the upcoming Dencun release.**
> 
> More info here: https://github.com/sigp/lighthouse/releases/tag/v5.0.0. TL;DR:
> 
> > All Lighthouse Beacon Nodes and Validator Clients must be upgraded to a v5.x.x release to ensure they follow the correct chain.
> 
> As it's my first PR to this repo, keeping it very light. We've modified these base Docker Compose files in other ways internally, but this seems like a critical fix that should be included as a sane default in the base files before the imminent Dencun upgrade.

Thanks to your issue, and it really helps a lot.

File: etc/lighthouse.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ name: reth
 services:
   lighthouse:
     restart: unless-stopped
-    image: sigp/lighthouse
+    image: sigp/lighthouse:v5.0.0-modern
```
Comment (onbjerg) [inline etc/lighthouse.yml:7]:
```suggestion
    image: sigp/lighthouse:v5.1.3-modern
```
updated

**Review by onbjerg — APPROVED** [review]

**Pull request merged**